src/OpalCompiler-Core/OCIRSend.class.st

```diff
@@ -18,6 +18,12 @@ OCIRSend >> accept: aVisitor [
 	^ aVisitor visitSend: self
 ]
 
+{ #category : 'stack' }
+OCIRSend >> deltaStack [
+
+	^ (selector occurrencesOf: $:) negated
+]
```

Comment:
We should think about if OCIRSend should not betterknow the number of arguments of the send (it could be set once when we create it). Bu this is a future improvement.